### DISCUSSION TITLE
Need to separate source code from project directory

### DISCUSSION THREAD

**Discussion by theremotewizard** [community/community#140886] [New to GitHub]
Hello everyone!

I have a directory of my project in GH. The directory contains the source code, various scripts, other files, and the executables but now i need to remove the source code files from the directory. Is there a way to do this and keep the source code in the same repo?

**Comment by buraito-kun** [comment]
You can head to your repository, then click on the directory or file that you want to delete

![image](https://github.com/user-attachments/assets/387eba99-011d-4ae9-9509-25a8002fce83)

1. Click on ...
2. Select Delete directory or file

May this will help you 🙂.

**Comment by theremotewizard** [comment]
Wouldn't deleting it remove it from the repo?

**Reply by buraito-kun** [reply]
Yes, delete from the repository in github, and don't forget to ignore your source code directory. Your source code on your PC won't be deleted.

**Reply by theremotewizard** [reply]
i would like to keep the source code separate from the directory. is there a way to move it out of the directory but keep it in the same repo?

**Reply by buraito-kun** [reply]
Sorry for my misunderstanding. If you want to move out from the directory that you are working in. It can't do.

### solution
Create a new repository and create 2 directories, one of which contains your source code and the other contains your various scripts.
